src/sections/peers.tsx

```diff
@@ -46,40 +55,95 @@ function PeersPage() {
         Peers seen in the network ({peers?.announced?.length || 0}){' '}
         <button onClick={handleRefresh}>Refresh</button>
       </h2>
-
+      <button
+        disabled={
+          !peers?.announced || Object.keys(peers.announced).length === 0
+        }
+        onClick={() => {
+          if (peers?.announced) {
+            exportToCsv(
+              peers.announced.map((peer) => ({
+                peerId: peer.peerId,
+                quality: peer.quality,
+                multiAddr: peer.multiAddr,
+                heartbeats: peer.heartbeats,
+                lastSeen: peer.lastSeen,
+                backoff: peer.backoff,
+                isNew: peer.isNew,
+              })),
+              'peers.csv'
+            );
+          }
+        }}
+      >
+        export
+      </button>
       <TableContainer component={Paper}>
         <Table sx={{ minWidth: 650 }} aria-label="aliases table">
           <TableHead>
             <TableRow>
-              <TableCell>id</TableCell>
-              <TableCell>peerId</TableCell>
-              <TableCell>alias</TableCell>
-              <TableCell>Elegible</TableCell>
-              <TableCell>Multiaddrs</TableCell>
+              <StyledTableCell sx={{ fontSize: '12px' }}>Id</StyledTableCell>
+              <StyledTableCell>Peer Id</StyledTableCell>
+              <StyledTableCell>Alias</StyledTableCell>
+              <StyledTableCell>Elegible</StyledTableCell>
+              <StyledTableCell>Multiaddrs</StyledTableCell>
+              <StyledTableCell>Last seen</StyledTableCell>
+              <StyledTableCell>Quality</StyledTableCell>
             </TableRow>
           </TableHead>
           <TableBody>
-            {Object.entries(peers?.announced ?? {}).map(([id, peer], key) => (
+            {Object.entries(peers?.announced ?? {}).map(([id, peer]) => (
               <TableRow key={id}>
-                <TableCell component="th" scope="row">
+                <StyledTableCell component="th" scope="row">
                   {id}
-                </TableCell>
-                <TableCell>{peer.peerId}</TableCell>
-                <TableCell>{getAliasByPeerId(peer.peerId)}</TableCell>
-                <TableCell>
+                </StyledTableCell>
+                <StyledTableCell>{peer.peerId}</StyledTableCell>
+                <StyledTableCell>
+                  {getAliasByPeerId(peer.peerId)}
+                </StyledTableCell>
+                <StyledTableCell>
                   {peers?.connected
                     .some(
                       (connectedPeer) => connectedPeer.peerId === peer.peerId
                     )
                     .toString()}
-                </TableCell>
-                <TableCell>{peer.multiAddr}</TableCell>
+                </StyledTableCell>
+                <StyledTableCell>
+                  {splitString(peer.multiAddr).map((address) => (
+                    <>
+                      {address}
+                      <br />
```

Comment:
why the <br/>?
```suggestion
```

Reply:
This line break is to show the addresses separated
Should we show them as one string?